runtime/backtrace.c

```diff
@@ -100,28 +100,35 @@ static void print_location(struct caml_loc_info * li, int index)
   }
 }
 
-/* Print a backtrace */
-CAMLexport void caml_print_exception_backtrace(void)
+/* Print a callstack backtrace */
+void caml_print_raw_backtrace(int len, backtrace_slot * buffer)
```

Comment:
I apologize for the nitpick, but I don't think the name of this function fits with its interface. I would expect `caml_print_raw_backtrace` to print a `raw_backtrace` value. I would suggest renaming it, for example `caml_print_backtrace_buffer`, and also passing the `int len` *after* the array, as I believe is more idiomatic in C.

(Otherwise the new version is fine, it nicely avoids any code duplication.)

Reply:
Done, thanks for the review.